### DISCUSSION TITLE
Column Filters

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#2341] [General] [answered]
Hi! I'm wondering if the column selector below (from Monkey) is also the default in Paste's table component? 

<img width="287" alt="Screen Shot 2022-04-05 at 1 58 11 PM" src="https://user-images.githubusercontent.com/87141551/161838914-503dca34-fb41-4759-8342-7da9ba1a5666.png">

**Comment by TheSisb** [comment, marked as answer]
Hi @ceyerkes,

At this time the Table component does not have prescribed column visibility toggles. This appears to be a custom solution created by that team.

**Reply by ceyerkes** [reply]
ok, thanks!
